remote_write.go

```diff
@@ -261,6 +262,10 @@ func (c *Client) send(state *lib.State, req []byte) (httpext.Response, error) {
 		r.Header.Set("X-Scope-OrgID", c.cfg.TenantName)
 	}
 
+	for k, v := range c.cfg.Headers {
+		r.Header.Set(k, v)
+	}
```

Comment:
personally, I'd put this before the `r.Header.Set("X-Scope-OrgID", c.cfg.TenantName)` so that if the custom headers also have an `X-Scope-OrgID` entry the one which is configured via the dedicated config property wins.

Reply:
yeah, makes sense